migrations/V1__my_first_migration.sql

```diff
@@ -0,0 +1,4 @@
+create table PERSON (
```

Comment:
Can we describe what this migration is doing in the filename? `V_1_create_table_person.sql` maybe?